raft-tech/cfgov-refresh
# added conditional copy for job

Client wanted a description for the add/edit screen for 'Job' that is slightly different from the descriptions for other incomes and expenses.  Completes #205 in cfgov/money-tools project board.


## Additions

- Added a couple of lines of conditional rendering for _Job_ description on _form.js_ file.
- Commented out _Category.description_ (not the same as the description above.  The _Category.description_ is used in a couple of the categories to provide additional information) for the 'Job' category in the _categories.js_ file.


## How to test this PR

1. Clear app cache and open.  Navigate to the calendar (no need to add starting balance).
2.  Tap + from _Bottom Nav_ and navigate to  'Job' transaction.  Note that the description reads as shown below in the screenshot.  Tap _Back_.
3. Tap + from _Bottom Nav_ and navigate to one other _Income_ transaction.  Note that the description reads as shown below in the second screenshot. Tap _Back_.
4.  Repeat above with and _Expense_ transaction.

## Screenshots

<img width="200" alt="Screen Shot 2020-08-12 at 6 04 12 PM" src="https://user-images.githubusercontent.com/34319929/90072954-53747d80-dcc6-11ea-8350-2f7b6e5c6158.png">
<img width="200" alt="Screen Shot 2020-08-12 at 6 04 32 PM" src="https://user-images.githubusercontent.com/34319929/90072953-52dbe700-dcc6-11ea-9557-89da43c70268.png">
